pyaedt/emit.py

```diff
@@ -117,6 +196,266 @@ def __init__(
+        ----------
+        >>> aedtapp.load_revision(path)
+
+        """
+        self._emit_api.load_result(path)
+        self._result_loaded = True
+    def result_mode_error():
+        """ prints error message"""
+        print("This function is inaccessible when the Emit object has no revisions")
+    @pyaedt_function_handler()
+    def interaction_domain():
+        """
+        Return a generic interaction domain.
+
+        Parameters
+        ----------
+       
+        Returns
+        -------
+        :class:`InteractionDomain`
+
+        References
+        ----------
+        >>> Emit.interaction_domain()
+        """
+        interaction_domain = EmitApiPython.InteractionDomain()
+        return interaction_domain
+    @pyaedt_function_handler()
+    def result_type(mode =''): 
+        """
+        Return a result_type object or a specific type based on input.
+
+        Parameters
+        ----------
+        mode:
+            Type of result_type object, options include: "emi", "desense", "sensitivity". Default is ''.
+
+        Returns
+        -------
+        :class:`result_type`
+
+        References
+        ----------
+        >>> Emit.result_type("emi")
+
+        """
+        result_t = EmitApiPython.result_type
+        if mode == 'emi':
+            result_t = EmitApiPython.result_type.emi
+        elif mode == 'desense':
+             result_t = EmitApiPython.result_type.desense
+        elif mode == 'sensitivity':
+             result_t = EmitApiPython.result_type.sensitivity 
+        return result_t
+    @pyaedt_function_handler()
+    def tx_rx_mode(mode = ''): 
+        """
+        Return tx_rx_mode object or a specific mode based on input.
+
+        Parameters
+        ----------
+        mode: str, option
+            Type of tx_rx_mode object, options include: "tx", "rx". Default is ''.
+
+        Returns
+        -------
+        :class:`tx_rx_mode`
+
+        References
+        ----------
+        >>> Emit.tx_rx_mode("tx")
+
+        """
+        tx_rx_m = EmitApiPython.tx_rx_mode
+        if mode == 'tx':
+            tx_rx_m = EmitApiPython.tx_rx_mode.tx
+        elif mode == 'rx':
+             tx_rx_m = EmitApiPython.tx_rx_mode.rx
+        return tx_rx_m
+    @pyaedt_function_handler()   
+    def copyright(self):
+        """
+        Return copyright info.
+
+        Parameters
+        ----------
+
+        Returns
+        -------
+        :class:`str`
+
+        References
+        ----------
+        >>> print(aedtapp.copyright())
+
+        """
+        copyright = self._emit_api.copyright()
+        return copyright
+    @pyaedt_function_handler()
+    def version(self, detailed = False):
+        """
+        Return version info.
+
+        Parameters
+        ----------
+        detailed: bool, optional
+            verbose or not, default is False.
+        Returns
+        -------
+        :class:`str`
+
+        References
+        ----------
+        >>> print(aedtapp.version())
```

Comment:
Same question here.

Reply:
However, I'm not sure I understood this comment, aedtapp.version() returns a string.